javatests/google/registry/rde/imports/XjcToDomainResourceConverterTest.java

```diff
@@ -120,6 +120,8 @@ public void testConvertDomainResource() throws Exception {
     assertThat(domain.getLastEppUpdateTime()).isNull();
     assertThat(domain.getAutorenewBillingEvent()).isNotNull();
     assertThat(domain.getAutorenewPollMessage()).isNotNull();
+    assertThat(domain.getAuthInfo()).isNotNull();
+    assertThat(domain.getAuthInfo().getPw().getValue()).hasLength(16);
```

Comment:
I don't think that will work, unless the string generator was injected somehow. I could modify the signature of the convert method to accept a string generator, which would work, but I'm hesitant to do so just for this.